scripts/bump-version.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+set -e
+
+# Needs to be run from the root directory of the project
+pip install bumpversion
```

Comment:
we may want to consider using bump2version which is a maintained fork of bumpversion and supports annotated git tags.